lib/generators/flipper/setup_generator.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails/generators/active_record'
+
+module Flipper
+  module Generators
+    class SetupGenerator < ::Rails::Generators::Base
+      desc 'Peform any necessary steps to install Flipper'
+
+      class_option :token, type: :string, default: nil, aliases: '-t',
+        desc: "Your Flipper Cloud development token"
```

Comment:
I'd maybe swap "development" to "personal", but if you feel strongly either way I'll leave it up to you.

Reply:
And probably explicitly state "...your personal _environment_ token"

Reply:
👍 Done